Classes/APAvatarImageView.m

```diff
@@ -8,19 +8,13 @@
 
 #import "APAvatarImageView.h"
 
-@interface APAvatarImageView ()
-
-- (void)draw;
```

Comment:
We need the interface as we are calling draw on line 72 before it is defined at the end of this file https://github.com/ankurp/APAvatarImageView/pull/6/files#diff-c8e8038627389230f306e921df1351c3R80